.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
 
 before_install:
   - sudo apt-get -qq update
-  - sudo apt-get install -y avahi-daemon libavahi-client-dev libavahi-common-dev libavahi-common3 libdbus-1-dev libupnp-dev libespeak-dev
+  - sudo apt-get install -y avahi-daemon libavahi-client-dev libavahi-common-dev libavahi-common3 libdbus-1-dev libupnp-dev libespeak-dev libudev-dev
```

Comment:
I am afraid that libudev is not available on OS X (or I couldn't find it), so we'll need to disable this adapter for OS X builds for now.